tracer/dependabot/Datadog.Dependabot.Integrations.csproj

```diff
@@ -238,12 +238,12 @@
 
     <!-- Integration: xunit.execution.desktop -->
     <!-- Assembly: xunit.execution.desktop -->
-    <!-- Latest package https://www.nuget.org/packages/xunit/2.5.3 -->
+    <!-- Latest package https://www.nuget.org/packages/xunit/2.6.0 -->
     <PackageReference Include="xunit" Version="2.5.3" />
```

Comment:
They haven't released a 2.6.0 of xunit.extensibility.vstudio yet, so bumping this breaks CI. Hopefully they release it soon otherwise we'll need to take another look at the tests..